main/FatesInventoryInitMod.F90

```diff
@@ -1267,8 +1243,8 @@ subroutine write_inventory_type1(currentSite)
        open(unit=pss_file_out,file=trim(pss_name_out), status='UNKNOWN',action='WRITE',form='FORMATTED')
        open(unit=css_file_out,file=trim(css_name_out), status='UNKNOWN',action='WRITE',form='FORMATTED')
 
-       write(pss_file_out,*) 'time patch trk age area water fsc stsc stsl ssc psc msn fsn'
-       write(css_file_out,*) 'time patch cohort dbh height pft nplant bdead alive Avgrg'
+       write(pss_file_out,*) 'time patch trk age area'
+       write(css_file_out,*) 'time patch cohort dbh height pft nplant'
```

Comment:
I guess if  we decide to delete `c_name` from above, then we ought to delete `cohort` from here.

Reply:
The review options don't let me comment on other lines, but we'd need to change the additional lines of code below:

```fortran
-           write(pss_file_out,*) '0000 ',trim(patch_str),' 2 ',currentPatch%age,currentPatch%area/AREA, &
-                 '0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000'
+           write(pss_file_out,*) '0000 ',trim(patch_str),' 2 ',currentPatch%age,currentPatch%area/AREA
```

and 

```fortran
-               write(cohort_str,'(A7,i4.4,A)') '<coh_',icohort,'>'
-               write(css_file_out,*) '0000 ',trim(patch_str),' ',trim(cohort_str), &
-                     currentCohort%dbh,0.0,currentCohort%pft,currentCohort%n/currentPatch%area,0.0,0.0,0.0
+               write(css_file_out,*) '0000 ',trim(patch_str),' ',&
+                     currentCohort%dbh,0.0,currentCohort%pft,currentCohort%n/currentPatch%area

```

(or keeping `cohort_str` if we decide to keep the cohort index).

Reply:
Oh, good catch! I've removed the unused columns from the write sections. Thanks @mpaiao !